pylint.cfg

```diff
@@ -1,2 +1,2 @@
 [MESSAGES CONTROL]
-disable=C0103,E1101,I1101,R0801
+disable=C0103,E1101,I1101,R0801,E0606,W0719
```

Comment:
Remove `W0719` (see comment in script below). If you absolutely want to catch broad `Exception`s, ignore the issue in line, not globally in the Pylint config. It is really not good practice to catch broad exceptions and therefore should only be used in rare cases. Disabling the check globally therefore invites bad practice.